addons/web/__manifest__.py

```diff
@@ -154,6 +154,12 @@
             # Don't include dark mode files in light mode
             ('remove', 'web/static/src/**/*.dark.scss'),
         ],
+        'web.assets_web': [
+            ('include', 'web.assets_backend'),
+            'web/static/src/main.js',
+            'web/static/src/legacy/legacy_setup.js',
+            'web/static/src/start.js',
```

Comment:
could be in assets_backend actually, it has no side-effect

Reply:
shouldn't this be in webclient anyway?